### PR TITLE
[Tri 414] network of network

### PR DESCRIPTION
**Pull request by mkanal**

### CODE REVIEW

File: uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+participant "EDCProvider" as EDCProvider #99FF99
+participant ESRCertificateStatisticSubmodelServerTier1 as ESRSubServerTier1
+end box
+
+User --> ESRSubServer: ../esr-certificate
+note left
+params: {globalAssetId, bomLifecycle, certificateName}
+end note
+ESRSubServer -> IRS : POST /irs/jobs
+ESRSubServer <-- IRS : 200
```
Comment (jzbmw) [inline uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml:26]: One line below

File: uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+note left
+params: {globalAssetId, bomLifecycle, depth=1}
+end note
+ref over IRS, DTRegistry, DAPS, "SubmodelServer AssemblyPartRelationship)"
+    Process JobResponse for given params
+end ref
+loop 100 times
```
Comment (jzbmw) [inline uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml:33]: why a 100 times? This loop is a loop until the result of the job is completed, correct. More like a polling

File: uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+       BPNL of Supplier
+   end note
+
+ESRSubServer -->  OnSupplyAPI : call OnSupplyAPI 
+note left
+    http//supplyOn/BPML_R/BPML_S/submodels/ISO14001
+    BPML_R : Requestor
+    BPML_S : Supplier
+end note
+ESRSubServer <--  OnSupplyAPI : ISO14001 Response
```
Comment (jzbmw) [inline uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml:66]: SupplyOn

File: uml-diagrams/spike/esr-certificates/esr-certificate-state-machine.puml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+@startuml
+
```
Comment (jzbmw) [inline uml-diagrams/spike/esr-certificates/esr-certificate-state-machine.puml:2]: Isnt the approval as of Monday out of scope?

**Review by jzbmw — CHANGES_REQUESTED** [review]

File: uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml (hunk 2 above)
Comment (mkanal) [reply uml-diagrams/spike/esr-certificates/esr-certificate-sequence-traverse_childs.puml:33]: Should have a predefined end. Endless looping willl cause in a job which could never gibve a result. 100 might a good number.

**Comment by mkanal** [comment]
@jzbmw Issues are fixed. Please review.

**Review by bubbur86 — APPROVED** [review]
#``

**Review by bubbur86 — COMMENTED** [review]
#331

**Review by jzbmw — APPROVED** [review]

**Comment by ds-jkreutzfeld** [comment]
@mkanal what is the state of this PR?

**Pull request merged**